snappi_ixnetwork/ixnetworkapi.py

```diff
@@ -362,6 +364,14 @@ def _connect(self):
                 self.warning('{}'.format(e))
         self._backup_errors()
 
+    def _ixn_version_check(self):
+        major, minor = self._globals.BuildNumber.split('.')[0:2]
+        if int(major) < 9:
+            return False
+        if int(major) >= 9 and int(minor) < 10:
```

Comment:
Won't this raise error for version `10.8` ?

Reply:
int(major) == 9 and int(minor) < 10